main/core/src/mill/internal/Utils.scala

```diff
@@ -0,0 +1,24 @@
+package mill.internal
+
+@mill.api.internal
+private[mill] object Utils {
```

Comment:
We already have more than one Uitl(s) class. Can you come up with another name (or prefix) to make reading the code more non-ambiguous? E.g. `AmmoniteUtils`.

Reply:
Done ✅